site/content/docs/gsoc-2020-ideas.md

```diff
@@ -61,6 +64,8 @@ engineers to implement new features. e.g. it may be interesting to extend the
 vendor tool to allow the extraction of a specific piece of IP from the
 OpenTitan repository and its dependencies.
 
+Mentors: Sam, Silvestrs
```

Comment:
Sound as good as long as the hardware guys don't mind me bombarding them with verilog related questions. Specifically in regard to:
```
This necessarily involves porting the core, making sure it fits and reasonably
maps. Then adding memory, timer, UART, SPI (leveraging + adapting existing
￼IP).
```

Reply:
We will definitely support you. Especially if the student is good, we want to make sure to maximize the output for the project and us, you won't be alone ;-)